imagetest/test_suites/ssh/image_ssh_test.go

```diff
@@ -0,0 +1,98 @@
+package ssh
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+	"golang.org/x/crypto/ssh"
+)
+
+var (
+	runtest = flag.Bool("runtest", false, "really run the test")
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+	if *runtest {
+		os.Exit(m.Run())
+	} else {
+		os.Exit(0)
+	}
+}
+
+func TestEmptyTest(t *testing.T) {
+	t.Logf("SSH target boot succesfully")
+	time.Sleep(600 * time.Second)
```

Comment:
i don't think this is needed - but you can test to be sure. either way this will make the test always take a minimum of 10 minutes